ml-data-hub-plugin/src/main/groovy/com/marklogic/gradle/exception/RetainDurationRequiredException.groovy

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright (c) 2021 MarkLogic Corporation
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ */
+
+package com.marklogic.gradle.exception
+
+import org.gradle.api.GradleException
+
+class RetainDurationRequiredException extends GradleException {
```

Comment:
No need for this class; the Java code can just throw an IllegalArgumentException